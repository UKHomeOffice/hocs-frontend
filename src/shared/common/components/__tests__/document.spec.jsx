import React from 'react';
import Document from '../document.jsx';

describe('Document preview component', () => {

    it('should render with default props', () => {
        const wrapper = render(<Document activeDocument={'MOCK_DOCUMENT_ID'} caseId={'MOCK_CASE_ID'} />);
        expect(wrapper).toBeDefined();
        expect(wrapper).toMatchSnapshot();
    });

});