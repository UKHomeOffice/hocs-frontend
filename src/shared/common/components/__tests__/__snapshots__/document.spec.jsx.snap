// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Document preview component should render with default props 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="document-preview"
      >
        <embed
          alt="pdf"
          height="100%"
          id="document-MOCK_DOCUMENT_ID"
          pluginspage="http://www.adobe.com/products/acrobat/readstep2.html"
          src="/case/MOCK_CASE_ID/document/MOCK_DOCUMENT_ID/preview"
          type="application/pdf"
          width="100%"
        />
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="document-preview"
    >
      <embed
        alt="pdf"
        height="100%"
        id="document-MOCK_DOCUMENT_ID"
        pluginspage="http://www.adobe.com/products/acrobat/readstep2.html"
        src="/case/MOCK_CASE_ID/document/MOCK_DOCUMENT_ID/preview"
        type="application/pdf"
        width="100%"
      />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
